### DISCUSSION TITLE
Loss of access to access tokens

### DISCUSSION THREAD

**Discussion by Uzkjzjjzk** [community/community#55244] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Select Feature Area

- [X] Issues
- [X] Projects

### Body

I lost access tokens and two-factor authentication, what do I do?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Uzkjzjjzk ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. 

However, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by Uzkjzjjzk** [reply]
Thank you for your reply. Unfortunately, I have searched more than one ticket, but to no avail
